paasta_tools/api/views/instance.py

```diff
@@ -521,6 +590,9 @@ def instance_status(request):
     include_smartstack = request.swagger_data.get("include_smartstack")
     if include_smartstack is None:
         include_smartstack = True
+    include_envoy = request.swagger_data.get("include_envoy")
+    if include_envoy is None:
+        include_envoy = True
```

Comment:
```
include_envoy = request.swagger_data.get("include_envoy", True)
```
?

Reply:
Unfortunately we can't do that because the key actually exists in the `swagger_data` dict with a `None` value.

```
>>> test = {'somekey': None}
>>> print(test.get('somekey'))
None
>>> print(test.get('somekey', True))
None
```